From 3c4b90c60ae6f48ff9cdb0603386bb2d67ab35b2 Mon Sep 17 00:00:00 2001
From: Phil Sutter <psutter@redhat.com>
Date: Wed, 13 Jan 2016 18:27:59 +0100
Subject: [PATCH] Documentation fixes

These changes will either be sent upstream soon or are rather Red Hat /
Fedora specific:

* Drop note that policy rule 0 can't be deleted.
* Create cbq.8 as a link to tc-cbq.8.
* Make ip-route.8 a little more consistent.
* Drop 'Debian' from ss.8, the information applies to Red Hat / Fedora
  as well.
* Refer to ss.ps instead of ss.html in ss.8, we don't ship the HTML
  variants.
---
 doc/ip-cref.tex        | 3 ---
 man/man8/cbq.8         | 1 +
 man/man8/ip-route.8.in | 2 +-
 man/man8/ip-rule.8     | 2 --
 man/man8/ss.8          | 4 ++--
 5 files changed, 4 insertions(+), 8 deletions(-)
 create mode 100644 man/man8/cbq.8

diff --git a/doc/ip-cref.tex b/doc/ip-cref.tex
index 67094c95b3681..242cc266b7acd 100644
--- a/doc/ip-cref.tex
+++ b/doc/ip-cref.tex
@@ -2049,9 +2049,6 @@ table \verb|local| (ID 255).
 The \verb|local| table is a special routing table containing
 high priority control routes for local and broadcast addresses.
 
-Rule 0 is special. It cannot be deleted or overridden.
-
-
 \item Priority: 32766, Selector: match anything, Action: lookup routing
 table \verb|main| (ID 254).
 The \verb|main| table is the normal routing table containing all non-policy
diff --git a/man/man8/cbq.8 b/man/man8/cbq.8
new file mode 100644
index 0000000000000..78940b6ac877c
--- /dev/null
+++ b/man/man8/cbq.8
@@ -0,0 +1 @@
+.so man8/tc-cbq.8
\ No newline at end of file
diff --git a/man/man8/ip-route.8.in b/man/man8/ip-route.8.in
index 9934a1e8c6f9a..bd1bb6969172a 100644
--- a/man/man8/ip-route.8.in
+++ b/man/man8/ip-route.8.in
@@ -16,7 +16,7 @@ ip-route \- routing table management
 
 .ti -8
 .BR "ip route" " { "
-.BR list " | " flush " } "
+.BR show " | " flush " } "
 .I  SELECTOR
 
 .ti -8
diff --git a/man/man8/ip-rule.8 b/man/man8/ip-rule.8
index b7008c6ab13d7..9b2c6b8704c36 100644
--- a/man/man8/ip-rule.8
+++ b/man/man8/ip-rule.8
@@ -111,8 +111,6 @@ The
 .B local
 table is a special routing table containing
 high priority control routes for local and broadcast addresses.
-.sp
-Rule 0 is special. It cannot be deleted or overridden.
 
 .TP
 2.
diff --git a/man/man8/ss.8 b/man/man8/ss.8
index f4d5264f30363..1116a62646888 100644
--- a/man/man8/ss.8
+++ b/man/man8/ss.8
@@ -131,7 +131,7 @@ Read filter information from FILE.
 Each line of FILE is interpreted like single command line option. If FILE is - stdin is used.
 .TP
 .B FILTER := [ state STATE-FILTER ] [ EXPRESSION ]
-Please take a look at the official documentation (Debian package iproute-doc) for details regarding filters.
+Please take a look at the official documentation (package iproute\-doc) for details regarding filters.
 
 .SH STATE-FILTER
 
@@ -186,7 +186,7 @@ Find all local processes connected to X server.
 List all the tcp sockets in state FIN-WAIT-1 for our apache to network 193.233.7/24 and look at their timers.
 .SH SEE ALSO
 .BR ip (8),
-.BR /usr/share/doc/iproute-doc/ss.html " (package iproute­doc)",
+.BR /usr/share/doc/iproute-doc/ss.ps " (package iproute\-doc)",
 .br
 .BR RFC " 793 "
 - https://tools.ietf.org/rfc/rfc793.txt (TCP states)
-- 
2.6.4

